docs/sphinx/ome-tiff/data.rst

```diff
@@ -98,6 +98,47 @@ section of the OME Data Model, derived from the public
 
 See `Neumann B et al. (2010). Phenotypic profiling of the human genome by time-lapse microscopy reveals cell division genes. Nature 464(7289):721 <https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3108885/>`__.
 
+Sub-resolutions
+^^^^^^^^^^^^^^^
+
+A set of OME-TIFF datasets representative of the :ref:`ometiff_subresolutions`
+section of the OME-TIFF specification were derived from a few large X Y images.
+
+.. list-table::
+  :header-rows: 1
+
+  -  * Dataset
+     * Image dimensions (XYZCT)
+     * Notes
+     * Provenance
+     * Copyright
+
+  -  * :ometiff_downloads:`Leica-1.ome.tiff <sub-resolutions/Brightfield/Leica-1/Leica-1.ome.tiff>`
+     * 36832 × 38432 × 1 × 3 × 1
+     * |WSI|, 2 RGB images
+     * `OpenSlide <http://openslide.cs.cmu.edu/download/openslide-testdata/Leica/>`_
+     * `Public Domain <https://creativecommons.org/publicdomain/mark/1.0/>`_
+  -  * :ometiff_downloads:`Leica-2.ome.tiff <sub-resolutions/Brightfield/Leica-2/Leica-2.ome.tiff>`
+     * 39168 × 26048 × 1 × 3 × 1
+     * |WSI|, 5 RGB images
+     * `OpenSlide <http://openslide.cs.cmu.edu/download/openslide-testdata/Leica/>`_
+     * `Public Domain <https://creativecommons.org/publicdomain/mark/1.0/>`_
+  -  * :ometiff_downloads:`LuCa-7color_Scan1.ome.tiff <sub-resolutions/Fluorescence/LuCa-7color_Scan1.ome.tiff>`
+     * 24960 × 34560 × 1 × 5 × 1
+     * |WSI|, multi-channel, fluorescence
+     * `PerkinElmer <http://www.perkinelmer.com/>`_
+     * `CC-BY 4.0 <https://creativecommons.org/licenses/by/4.0>`_
+  -  * :ometiff_downloads:`BGal_000438_frames.ome.tiff <sub-resolutions/EM/BGal_000438_frames.ome.tiff>`
+     * 7676 × 7420 × 1 × 1 × 38
```

Comment:
Should be 7676 x 7420 x 38 x 1 x 1

From reading the file:
SizeC = 1
SizeT = 1
SizeX = 7676
SizeY = 7420
SizeZ = 38

Reply:
Fixed in ae6861d